kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/assertj/StatusAssertions.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kubernetes.operator.assertj;
+
+import io.kroxylicious.kubernetes.api.v1alpha1.KafkaProxyStatus;
+
+public class StatusAssertions {
```

Comment:
what's this classes purpose? Maybe it would make more sense if it was called `OperatorAssertions` and had `assertThat` methods for each of the `*Assert` classes, so we'd have a single class to discover them and it would follow assertj's pattern/naming.

Reply:
`OperatorAssertions` seems reasonable (we have `KafkaAssertions` elsewhere already)